### ISSUE TITLE
doc: remove info on perftune.yaml modes from production readiness guidelines

### ISSUE THREAD

**Issue by annastuchlik** [scylladb/scylladb#14873]
The [Production Readiness](https://opensource.docs.scylladb.com/stable/operating-scylla/procedures/tips/production-readiness.html) page includes information on perftune modes, which are **deprecated**:
![image](https://github.com/scylladb/scylladb/assets/37244380/86a99c76-1450-4639-baee-535139942ae3)

**Fix**
I think the information about `perftune.yaml` and `cpuset.conf `should be removed. Instead, a short recommendation to run perftune.py should be added.

@vladzcloudius Please confirm.

**Comment by annastuchlik** [comment]
@vladzcloudius 
Please have a look at https://opensource.docs.scylladb.com/stable/kb/perftune-modes-sync.html.
Perhaps this article is outdated, too?

**Comment by vladzcloudius** [comment]
> @vladzcloudius OK, so my understanding is that I should leave the article, remove the reference to the specific issue and `sq_split` mode, and add the information that it is a procedure to ensure that the perftune configuration is correct after a ScyllaDB upgrade. If that's the case, should we recommend the procedure to all ScyllaDB users who customize their perftune.py?

Yup. I think the same procedure applies. This would require us adjusting that procedure wording of course - to make it more generic.
And then we can reference it from both places.
